src/amo/components/RatingsByStar/index.js

```diff
@@ -86,85 +90,108 @@ export class RatingsByStarBase extends React.Component<InternalProps> {
     );
   }
 
+  handleRouting(star: string) {
+    const { addon, location, history, clientApp, siteLang } = this.props;
+    history.push({
+      pathname: `/${siteLang}/${clientApp}/addon/${addon.slug}/reviews/`,
+      search: `?score=${star}&${
```

Comment:
I think you need to put the `&` inside the `{ }` for `location.query.src` because the way the code is now, without a `src` you still get a trailing `&` on the URL.

Reply:
Actually while writing tests for it I found a better approach for doing this. It goes like this:
```
    history.push(
      `/${siteLang}/${clientApp}/${reviewListURL({
        addonSlug: addon.slug,
        score: star,
        src: location.query.src,
      })}`,
    );
```
which will use the general function for creating url.